src/components/body/Controller.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+
+export default function Controller({ onModeChange }) {
```

Comment:
next time instead of breaking down props object in the declaration like this use props and props.[features]